packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -84,19 +101,26 @@ export const TitleAndDatePickerComponent = ({
     });
   };
 
+  const handlePinChange = () => {
+    setPinnedOverlay(mapOverlayCode);
+  };
+
   return (
-    <>
-      <Content expanded={isExpanded} selected={isMapOverlaySelected} period={periodGranularity}>
-        <ContentText>{isMeasureLoading ? <CircularProgress size={22} /> : name}</ContentText>
-        <ToolsWrapper>
+    <Wrapper>
+      <Content>
+        <FlexSpaceBetween>
+          <PinWrapper>
```

Comment:
Instead of `PinWrapper` it would be cleaner to just style `Pin` directly.

Reply:
You could do that either by adding the style in Pin.js or by extending it in this file. If you extend it in this file you will need to add the className prop to the Pin component for styledComponents to work.

Reply:
Thanks for sharing this. Thought we always need to wrap it for the padding.